doc/web-socket.rst

```diff
@@ -0,0 +1,405 @@
+.. _web-socket:
+
+Web Socket
+==========
+
+Broker offers access to the publish/subscribe layer via WebSocket in order to
+make its :ref:`data <data-model>` accessible to third parties.
+
+WebSocket clients are treated as lightweight peers. Each Broker endpoint can be
+configured to act as a WebSocket server by either setting the environment
+variable ``BROKER_WEB_SOCKET_PORT`` or by setting ``broker.web-socket.port`` on
+the command line or in the configuration file. When running Zeek, scripts may
+also redefine ``Broker::web_socket_port`` to a valid TCP port.
```

Comment:
```suggestion
also redefine ``Broker::default_port_websocket`` to a valid TCP port.
```

That's how my Zeek-side branch now names this.

Reply:
I've changed this in [5b26424](https://github.com/zeek/broker/pull/219/commits/5b26424a823fb12cda46e9e009bb97e6ce97c793).